test/e2e/s2i-e2e.bats

```diff
@@ -18,7 +20,8 @@ declare -rx E2E_S2I_LANGUAGE="${E2E_S2I_LANGUAGE:-}"
     [ -n "${E2E_S2I_PARAMS_REVISION}" ]
     [ -n "${E2E_S2I_PARAMS_IMAGE}" ]
     [ -n "${E2E_PARAMS_TLS_VERIFY}" ]
```

Comment:
Given the `E2E_S2I_IMAGE_SCRIPTS_URL` is required, you need to check if it's defined as well, i.e.:

```bash
[ -n "${E2E_S2I_IMAGE_SCRIPTS_URL}" ]
```